src/CommandRunner/Git.php

```diff
@@ -61,11 +61,13 @@ public function getRemoteUrl(string $remote = 'origin'): string
      * Check if the git repository in the provided directory has any uncommitted changes
      *
      * @param string $directory A directory containing a git repository
+     *
```

Comment:
I don't see a reason to do this

Reply:
It is a CS fix to be coherent with the rest of the codebase (there is a new line before every `@return` everywhere else).